templates/back/signalement/view/header.html.twig

```diff
@@ -218,7 +218,7 @@
                             </button>
                         {% endif %}
                     {% endif %}
-                    {% if signalement.statut is not same as constant('App\\Entity\\Signalement::STATUS_CLOSED') or (is_granted('ROLE_ADMIN_TERRITORY') and isAffected) %}
+                    {% if signalement.statut is not same as constant('App\\Entity\\Signalement::STATUS_CLOSED') or (is_granted('ROLE_ADMIN_TERRITORY') or isAffected) %}

```

Comment:
Pour la réouverture RT pour moi le isAffected est trop 
Et j'ai du mal avec cette condition `if signalement.statut is not same as constant('App\\Entity\\Signalement::STATUS_CLOSED'`

La condition la plus clair pour moi c'est ça serait plutôt si le statut est refusé et qu'il est RT